### ISSUE TITLE
Add this to documentation

### ISSUE THREAD

**Issue by janswist** [twbs/icons#1646]
It took me an hour to find that there was a missing `import` statement in docs for icons to work with Vue:

```
import 'bootstrap-icons/font/bootstrap-icons.css'
```

Please, add it to the docs. I hope others won't waste so much time as I did.

**Comment by korki43** [comment]
I couldn't find anything about Vue in the documentation. Which part of the documentation are you referring to? The [usage](https://icons.getbootstrap.com/#usage) section of the documentation only shows how to 'include the icons into your HTML'.

In some JS libraries like Vue or React it is common practice to import CSS files in the JavaScript. This is something that is explained in the libraries documentation ([e.g. MDN](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Vue_styling)).

**Comment by janswist** [comment]
It's not in the Vue documentation. I found it as the only working solution for icons to actually work. Without it it just simply didn't showed anything.

Also this:
![image](https://user-images.githubusercontent.com/5583954/227789720-ac37070e-9f40-40f0-9592-48f89d606fb9.png)
I found to be lacking extra `bi` class which also prevented icon from showing. Correct version that worked for me was `class="bi bi-alarm"`.

Source: https://icons.getbootstrap.com/#usage

**Comment by korki43** [comment]
The documentation explicitly says **'Include the icon web fonts in your page via CSS'**. If you build a HTML webpage this means adding a `<link>` element, but for other use cases this might mean other things (e.g. import statements in JS). You need to know how to use your tools, because it's impossible to document every single possibility in which Bootstrap icons might be used.

As for the missing `bi` class: I think the need for a `bi` class was removed in an earlier version. Maybe check if you are using the latest version of bootstrap-icons and update if needed.

Example Codepen: https://codepen.io/korki43/pen/ZEMwoPj

**Comment by janswist** [comment]
I installed `bootstrap-icons` yesterday for the first time (migrating from `bootstrap-vue`).

I recreated the problem here: https://stackblitz.com/edit/vue-v29fsr?file=src/main.js (unsuccessfully, so not sure now how it is working in my project and where it's taking icons from).

I based off of understanding that installing npm package (`npm i bootstrap-icons`) is sufficient so knowing how to use tool depends on the instructions it provides. A lot of developers use this framework (not sure if the newest one, tho) so it might probably prove helpful to many others to solve it.
